src/main/java/graphql/GraphQLError.java

```diff
@@ -39,8 +39,10 @@ public interface GraphQLError extends Serializable {
     ErrorClassification getErrorType();
 
     /**
-     * The graphql spec says that the (optional) path field of any error should be a list
-     * of path entries https://spec.graphql.org/October2021/#sec-Handling-Field-Errors
+     * The graphql spec says that the (optional) path field of any error must be
+     * a list of path entries starting at the root of the response
+     * and ending with the field associated with the error
+     * https://spec.graphql.org/draft/#sec-Errors.Error-Result-Format
```

Comment:
Unrelated to AST printing, there is a minor spec change coming soon that specifies the path (if present) must be a list from root to the field. As this engine is already compliant this is only a comment change https://github.com/graphql/graphql-spec/pull/1073